js/modules/xrange.src.js

```diff
@@ -75,8 +69,12 @@ seriesType('xrange', 'column', {
 		}
 	},
 	tooltip: {
-		headerFormat: '<span style="font-size: 0.85em">{point.x} - {point.x2}</span><br/>',
-		pointFormat: '<span style="color:{point.color}">\u25CF</span> {series.name}: <b>{point.yCategory}</b><br/>'
+		headerFormat:
```

Comment:
Multiline headerFormat and pointFormat defintions are not correctly picked up in the API - we must either fix that, or leave formats on one line.

Reply:
Resolved